pom.xml

```diff
@@ -59,7 +59,8 @@
     <!-- Change this property if you need your tests to be compiled with a different Java level than the plugin. -->
     <!-- Use only if strictly necessary. It may cause problems in your IDE. -->
     <java.level.test>${java.level}</java.level.test>
-    <concurrency>1</concurrency> <!-- may use e.g. 2C for 2 × (number of cores) -->
+    <concurrency>1</concurrency> <!-- DEPRECATED use forkCount -->
```

Comment:
not sure we even need this but it here for backwards compatability...  happy to remove both this and `forkCount` (as the default is one...)